app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :authenticate_api_v1_user!, only: %i[index create]
+      def index
+        render_paginated Rent.filter(params.slice(:user_id, :book_id))
+      end
+
+      def create
+        rent = Rent.new(
+          book_id: params[:book_id],
+          user_id: params[:user_id],
+          from: params[:from],
+          to: params[:to]
+        )
+        if rent.save
+          render json: rent
+        else render json: {}, status: :unprocessable_entity
```

Comment:
the indentation is a little confusing, try putting the `render...` in a new line.
BTW, you could return the errors with `rent.errors`